ynput/ayon-frontend
# Reviewables: Show play icon when entity has a reviewable

## Description of changes 
Show a little play icon when some entity has a reviewable to watch.
1. Kanban card
2. Entity Details panel
3. Browser card


## Testing

1. `yarn install`
2. Upload a reviewable for a version
3. Check that the play icon appears in the three locations above 
4. Check the folder of the version shows the icon in the details panel.


## Additional context

![image](https://github.com/user-attachments/assets/d4985877-eb7d-4184-969a-18b7f53cf9b7)
![image](https://github.com/user-attachments/assets/a00ad189-0cab-4c4f-bfc9-035693b268f3)
